_config.yml

```diff
@@ -118,6 +118,11 @@ nav_external_links:
   - title: Just the Docs on GitHub
     url: https://github.com/just-the-docs/just-the-docs
 
+liquid:
+   error_mode: strict
+   #strict_variables: true
```

Comment:
A nit, but I think we probably shouldn't include this, even as a comment? 

```suggestion
```

Reply:
Yeah, it's likely not a setting that'll ever be enabled so it's probably best to keep it away.